power_grid_model_c/power_grid_model/include/power_grid_model/optimizer/tap_position_optimizer.hpp

```diff
@@ -35,14 +38,165 @@ struct TrafoGraphEdge {
     EdgeWeight weight{};
```

Comment:
@Jerry-Jinfeng-Guo Let us use either WeightedTrafo or TrafoGraphEdge. Is there a reason to use WeightedTrafo?

Reply:
Indeed redundant, let's do:
```
using WeightedTrafoList = std::vector<TrafoGraphEdge>;
```

Reply:
A few places where `WeightedTrafo` were also used could use an update as well.